backend/src/test/java/com/github/sirtobiwan/backend/controllers/VaccineControllerTest.java

```diff
@@ -0,0 +1,60 @@
+package com.github.sirtobiwan.backend.controllers;
+
+import com.github.sirtobiwan.backend.models.Vaccine;
+import com.github.sirtobiwan.backend.models.VaccineRepo;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import java.time.LocalDate;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class VaccineControllerTest {
+
+    @Autowired
+    MockMvc mockMvc;
+
+    @Autowired
+    VaccineRepo vaccineRepo;
+
+    @Test
+    void expectAllVaccines_whenGetRequestForAllVaccines() throws Exception {
+        Vaccine newVaccine = new Vaccine("123", "Corona", "Biontech", "2", LocalDate.of(2023, 07, 25), "Dr. Meier", true, LocalDate.of(2024, 07, 25));
+        Vaccine newVaccine2 = new Vaccine("456", "Corona", "Johnson", "2", LocalDate.of(2023, 07, 25), "Dr. Meier", true, LocalDate.of(2024, 07, 25));
+        vaccineRepo.save(newVaccine);
+        vaccineRepo.insert(newVaccine2);
```

Comment:
Why did you use insert?

Reply:
just out of curiosity ;) Same result. Thanks